mailbox/event/json/src/test/java/org/apache/james/event/json/MailboxDeletionSerializationTest.java

```diff
@@ -55,6 +56,7 @@
         SESSION_ID,
         USERNAME,
         MAILBOX_PATH,
+        new MailboxACL(),
```

Comment:
Can you add a test case ensuring we can still de-serialize an event when the JSON is missing the ACL field? 

This will avoid us a breaking change!